docs/modules/operation/pages/events/sources/kafka.adoc

```diff
@@ -0,0 +1,50 @@
+
+[[ga-events-sources-kafka]]
+
+= Kafka Consumer (Events)
+
+Events can also be created by routing them to a specific topic on Kafka.
+The Kafka Consumer feature allows events to be consumed from specific topic on Kafka.
+
+Events should be created in protobuf format link:https://developers.google.com/protocol-buffers/[Google Protocol Buffers (GPB)]
```

Comment:
Create events in protobuf fomrat link:https://developers.google.com/protocol-buffers/[Google Protocol Buffers (GPB)].